package.json

```diff
@@ -15,6 +15,7 @@
     "lint": "eslint lib test"
   },
   "dependencies": {
+    "boolean": "^0.1.1",
```

Comment:
This dependency on [`boolean`](https://www.npmjs.com/package/boolean) is added so `filter[field][exists]` can take `"true"`, `"false"`, `"1"`, `"0"` and so on.